src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/system_request_test.cc

```diff
@@ -229,6 +229,125 @@ TEST_F(SystemRequestTest, Run_RequestTypeDisallowed_SendDisallowedResponse) {
   command->Run();
 }
 
+TEST_F(SystemRequestTest, Run_FileWithBinaryDataSaveUnsuccessful_GenericError) {
+  std::vector<uint8_t> binary_data;
```

Comment:
@ZhdanovP Don't we have typedef fot this type if data?

Reply:
@AByzhynar please take a look 8ed4a05b95d7bfc885e2bad85a6970636ef2e7f8